src/Tribe/Editor/Template/Overwrite.php

```diff
@@ -191,34 +194,40 @@ public function has_tickets_support( $post_id ) {
 	 * @param $post_id
 	 */
 	public function setup_template( $post_id ) {
-		tribe( 'tickets.editor.template' )->add_template_globals( array(
-			'post_id' => $post_id,
-			'post'    => get_post( $post_id ),
-		) );
+		/** @var Tribe__Tickets__Editor__Template $template */
+		$template = tribe( 'tickets.editor.template' );
+
+		$template->add_template_globals(
+			[
+				'post_id' => $post_id,
+				'post'    => get_post( $post_id ),
+			]
+		);
 	}
 
 	/**
-	 * Remove the actions and filters used to attach the classic editor views associated with
-	 * tickets
+	 * Remove the actions and filters used to attach the classic editor views associated with tickets.
 	 *
 	 * @since 4.9
 	 */
 	public function remove_classic_views() {
-		if ( ! ( $this->ticket_type instanceof Tribe__Tickets__Tickets ) ) {
+		if ( ! $this->ticket_type instanceof Tribe__Tickets__Tickets ) {
 			return;
 		}
 
 		$ticket_form_hook = $this->ticket_type->get_ticket_form_hook();
 		if ( ! empty( $ticket_form_hook ) ) {
-			remove_action( $ticket_form_hook, array(
+			remove_action(
+				$ticket_form_hook, [
 				$this->ticket_type,
 				'maybe_add_front_end_tickets_form',
-			), 5 );
+			], 5 );
 		}
 
-		remove_filter( 'the_content', array(
+		remove_filter(
+			'the_content', [
 			$this->ticket_type,
 			'front_end_tickets_form_in_content',
-		), 11 );
+		], 11 );
```

Comment:
* :no_entry_sign: Line indented incorrectly; expected at least 3 tabs, found 2
* :no_entry_sign: Array closer not aligned correctly; expected 12 space(s) but found 8
* :no_entry_sign: Only one argument is allowed per line in a multi-line function call
* :no_entry_sign: Closing parenthesis of a multi-line function call must be on a line by itself

<sub>hash: 981b3fe65eb29bc749752c759c5fdedd</sub>